website/src/react-native/css/react-native.css

```diff
@@ -2030,6 +2030,14 @@ article li {
 .btn:hover {
   text-decoration: none !important; }
 
+@media screen and (max-width: 373px) {
+  .btn {
```

Comment:
Should this be applying to *all* buttons on small screens or should this rule be `.btn.docs-prev, .btn.docs-next`? I haven't checked this PR locally to verify buttons across the site look fine on small screens.

Also, you add margin-bottom in this rule for the stacked buttons, do we need your addition of margin-bottom (of a different amount) for `.buttons-unit .button` above?